stories/topics/proc-aws-vpc-peering.adoc

```diff
@@ -30,4 +30,4 @@ This can be the {PlatformNameShort} from {Market} VPC
 
 When you have configured the direct network peering you must configure the route tables, security groups, and Network ACLs for both VPCs for traffic to route properly.
 
-For additional information on route tables, security groups and other networking components regarding VPC Peering, see the link:https://docs.aws.amazon.com/vpc/latest/peering/what-is-vpc-peering.html[AWS documentation].
+For additional information on route tables, security groups and other networking components regarding VPC Peering, see the link:https://docs.aws.amazon.com/vpc/latest/peering/what-is-vpc-peering.html[What is VPC peering?].
```

Comment:
, see link:https://docs.aws.amazon.com/vpc/latest/peering/what-is-vpc-peering.html[What is VPC peering?] in the AWS documentation.